src/configs/astro.ts

```diff
@@ -64,8 +78,38 @@ export async function astro(
             }
           : {},
 
+        ...formatter
+          ? DISABLE_STYLISTIC_RULES
+          : {},
+
         ...overrides,
       },
     },
+    {
+      // Define the configuration for `<script>` tag when using `client-side-ts` processor.
+      files: ['**/*.astro/*.ts', '*.astro/*.ts'],
+      languageOptions: {
+        globals: {
+          ...globals.browser,
+        },
+        parser: parserTs,
+        parserOptions: {
+          project: null,
+        },
+        sourceType: 'module',
+      },
+      // Script in `<script>` is assigned a virtual file name with the `.ts` extension.
+      name: 'antfu/astro/base/typescript',
+      rules: {
+        'prettier/prettier': 'off',
```

Comment:
We don't have the prettier plugin; we don't disable rules, and we are not aware of them.

Reply:
it's true, that line was old when i was thought that `format/prettier` was `prettier/prettier`